### ISSUE TITLE
dev: deploy l1 contracts on dockerized anvil node in local-rpc

### ISSUE THREAD

**Issue by enitrat** [kkrt-labs/kakarot-rpc#1285]
following https://github.com/kkrt-labs/kakarot-rpc/pull/1284 we have an anvil node that acts as the settlement layer used with Katana.

in .katana/messaging_config.json we hardcoded the address of the StarknetMessaging contract - but it would be better to, in the docker-compose, deploy the StarknetMessaging and L1KakarotMessaging contracts and put them in the .env to make it more robust.

**Comment by greged93** [comment]
@enitrat is this done?

**Comment by enitrat** [comment]
no, this is still relying on the dynamic addresses deployed on the anvil node. As long as the order of deployment and the key account used to deploy are the same, this is not an issue because they will always have the same addresses.

**Comment by Eikix** [comment]
what is the status of this @enitrat ?

**Comment by enitrat** [comment]
v low prio so nothing done
